thirdparty/build-thirdparty.sh

```diff
@@ -517,6 +517,27 @@ build_librdkafka() {
     make -j$PARALLEL && make install
 }
 
+# arrow
+build_arrow() {
+    check_if_source_exist $ARROW_SOURCE
+
+    cd $TP_SOURCE_DIR/$ARROW_SOURCE/cpp
+
+    mkdir release && cd release && \
+    cmake .. -DARROW_PARQUET=ON -DARROW_OPTIONAL_INSTALL=ON -DBOOST_ROOT=$TP_INSTALL_DIR && \
+    make -j$PARALLEL parquet && make install && \
+    cp -r /usr/local/include/arrow $TP_INSTALL_DIR/include/ && \
```

Comment:
you can use -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR to avoid copy install yourself